### PR TITLE
[feature] Rakeman main logic services

### PR DESCRIPTION
**Pull request by CaptainJNS**

### CODE REVIEW

File: app/services/rakeman/task_creator.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    Rakeman::RakeTask.destroy_all if destroy_old_tasks
+
+    tasks_list ||= tasks
+
+    tasks_list.each do |task|
+      Rakeman::RakeTask.create(name: task[:name], description: task[:description])
+    end
+  end
+
+  def tasks
```
Comment (CaptainJNS) [inline app/services/rakeman/task_creator.rb]: `all_tasks_list`

File: rakeman.gemspec [hunk 2]
```diff
@@ -1,32 +1,49 @@
...
 
 # Describe your gem and declare its dependencies:
 Gem::Specification.new do |spec|
-  spec.name          = "rakeman"
+  spec.name          = 'rakeman'
   spec.version       = Rakeman::VERSION
-  spec.authors       = ["CaptainJNS"]
-  spec.email         = ["sereda9623@gmail.com"]
+  spec.authors       = ['CaptainJNS']
+  spec.email         = ['sereda9623@gmail.com']
```
Comment (CaptainJNS) [inline rakeman.gemspec]: add others

File: rakeman.gemspec [hunk 3]
```diff
@@ -1,32 +1,49 @@
...
-      "public gem pushes."
+    raise 'RubyGems 2.0 or newer is required to protect against ' \
+      'public gem pushes.'
   end
 
-  spec.files = Dir["{app,config,db,lib}/**/*", "MIT-LICENSE", "Rakefile", "README.md"]
+  spec.files = Dir['{app,config,db,lib}/**/*', 'MIT-LICENSE', 'Rakefile', 'README.md']
+
+  spec.add_dependency 'rails', '>= 5.1.5'
+  spec.add_dependency 'rake', '~> 12.0'
```
Comment (CaptainJNS) [inline rakeman.gemspec]: do we need this?

Comment (CaptainJNS) [reply rakeman.gemspec]: nope

File: spec/dummy/config/locales/errors.yml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+en:
```
Comment (CaptainJNS) [inline spec/dummy/config/locales/errors.yml]: move to engine `/config/locales/en.yml`
also `en.rakeman.errors`

File: spec/support/config/capybara.rb [hunk 5]
```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'capybara/rails'
+require 'capybara/rspec'
+require 'selenium-webdriver'
+
+Capybara.register_driver(:chrome) do |app|
```
Comment (CaptainJNS) [inline spec/support/config/capybara.rb:7]: add feature tests

Comment (CaptainJNS) [reply spec/support/config/capybara.rb:7]: later

File: rakeman.gemspec [hunk 6]
```diff
@@ -9,8 +9,8 @@ require 'rakeman/version'
 Gem::Specification.new do |spec|
   spec.name          = 'rakeman'
   spec.version       = Rakeman::VERSION
-  spec.authors       = ['CaptainJNS']
-  spec.email         = ['sereda9623@gmail.com']
+  spec.authors       = %w[CaptainJNS olehvolynets YaroslavZahoruiko]
+  spec.email         = %w[sereda9623@gmail.com]
```
Comment (CaptainJNS) [inline rakeman.gemspec:13]: @olehvolynets @YaroslavZahoruiko add your emails please in separate PR

**Review by YaroslavZahoruiko — APPROVED** [review]

**Pull request merged**